src/lib/src/ng-select.component.ts

```diff
@@ -187,6 +192,9 @@ export class NgSelectComponent implements OnInit, OnDestroy, ControlValueAccesso
             } else {
                 this.selectWriteValue(value);
             }
+            if (this.single) {
+                this.filterValue = this.getLabelValue(this.value);
```

Comment:
Setting filterValue = label seems to not be work correctly with custom label template. It would be better to leave filterValue as is, because this is that user is typing, but I will test how this work to see it in real life.